.github/workflows/pr-pipeline.yml

```
@@ -0,0 +1,26 @@
+    with:
+      python_version: "3.13"
+      run_tests: true
+
+  pr-comment:
+    name: PR Summary
+    needs: build-test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Print PR Summary
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)